docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -110,6 +110,27 @@ Response
 :   @@snip [updated.json](assets/resources/updated.json)
 
 
+## Refresh
+
+This operation refreshes the payload.
```

Comment:
`This operation refreshes the compacted and expanded representations of the resource.`